src/test/java/de/tum/in/www1/artemis/ParticipationIntegrationTest.java

```diff
@@ -67,11 +87,21 @@ public void initTestCase() {
         }
         modelingExercise.setTitle("UML Class Diagram");
         exerciseRepo.save(modelingExercise);
+
+        programmingExercise = ModelFactory.generateProgrammingExercise(ZonedDateTime.now().minusDays(1), ZonedDateTime.now().plusDays(1), course);
+        programmingExercise = exerciseRepo.save(programmingExercise);
+        course.addExercises(programmingExercise);
+        course = courseRepo.save(course);
+
+        doReturn("Success").when(continuousIntegrationService).copyBuildPlan(any(), any(), any(), any(), any());
+        doNothing().when(continuousIntegrationService).configureBuildPlan(any());
+        doNothing().when(continuousIntegrationService).performEmptySetupCommit(any());
     }
 
     @AfterEach
     public void tearDown() {
         database.resetDatabase();
+        Feature.PROGRAMMING_EXERCISES.enable();
```

Comment:
good! this change is also needed in the other PR

Reply:
Added this to the other PR as well